ansto_simplon_api/schemas/configuration.py

```diff
@@ -62,7 +62,7 @@ class ZMQStartMessage(BaseModel):
     pixel_mask_enabled: bool = True
     pixel_size_x: float = 7.5e-05
     pixel_size_y: float = 7.5e-05
-    saturation_value: int | None = 0
+    saturation_value: int | None = 33000
```

Comment:
Can you add a FIXME or TODO here? 
I need to get back to where to get this magic number from.